pymc/backends/report.py

```diff
@@ -139,44 +148,23 @@ def _run_convergence_checks(self, idata: arviz.InferenceData, model):
 
         warnings = []
         rhat_max = max(val.max() for val in rhat.values())
-        if rhat_max > 1.4:
+        if rhat_max > 1.01:
             msg = (
-                "The rhat statistic is larger than 1.4 for some "
-                "parameters. The sampler did not converge."
-            )
-            warn = SamplerWarning(WarningType.CONVERGENCE, msg, "error", extra=rhat)
-            warnings.append(warn)
-        elif rhat_max > 1.2:
-            msg = "The rhat statistic is larger than 1.2 for some " "parameters."
-            warn = SamplerWarning(WarningType.CONVERGENCE, msg, "warn", extra=rhat)
-            warnings.append(warn)
-        elif rhat_max > 1.05:
-            msg = (
-                "The rhat statistic is larger than 1.05 for some "
-                "parameters. This indicates slight problems during "
-                "sampling."
+                "The rhat statistic is larger than 1.01 for some "
+                "parameters. This might indicate problems during sampling."
```

Comment:
are there cases with large rhat that have no problems during sampling? I think the might can be confusing and overly optimistic, but I am biased towards strong wording given how I have seen disastrous diagnostics be ignored very often

Reply:
My hope with these is to also give users a followup action to take, so a citation might help -- what do you think? 

With respect to "are there cases with large rhat that have no problems during sampling?", i did two quick experiments using a poorly behaved distribution (Cauchy), and independent samples. My rough impression is that in this case, it would be quite rare to get an rhat over 1.01, but this is a super sketchy experiment in a github comment, so who knows:
1. Simulate drawing 4 chains of 400 draws from a Cauchy(1), and check the max r-hat from doing that 1,000 times:
![image](https://user-images.githubusercontent.com/2295568/155454351-2eb20c9e-8250-4070-b5e9-3e8a0bc83b08.png)
2. What if we had shorter chains? Here is a plot of the rhat after `n` draws, where `n` goes from 30 to 400 total draws:
![image](https://user-images.githubusercontent.com/2295568/155454647-5af19a17-bf69-44c1-b885-1a56461a822a.png)

this one changes a lot depending on which draws you get, so here are 6 such experiments on the same plot: ![image](https://user-images.githubusercontent.com/2295568/155454525-50b80241-3e1e-4baa-aad8-a3640e2def4c.png)

Reply:
I swear that I was internally struggling with using or not that "might" :-) It was an attempt to soften that hard threshold. But agree that strong language is a better idea, users are still free to ignore warnings, but at least now they will feel a little bit more guilty, haha. 

I agree @ColCarroll that we should provide actions. We have been talking with Oriol, Aki and others about having a centralized website that we can point users to. This site could have "levels of information" so the first level could be a direct action, the second level could provide more nuanced recommendations and/or a better explanations of why the diagnostics work and a third level could be a paper.

Regarding your experiments for short chains at least too short the ESS will be low, so the r-hat is not trustworthy.